PrebidMobile/PrebidMobile-core/src/main/java/org/prebid/mobile/addendum/AdViewUtils.java

```diff
@@ -227,6 +230,53 @@ public void onReceiveValue(@Nullable String html) {
         });
     }
 
+    /**
+     * {@link PrebidNativeAdListener} will be called only once.
+     * {@link PrebidNativeAdListener#onPrebidNativeLoaded(PrebidNativeAd)} when Cached Ad is found and is valid
+     * {@link PrebidNativeAdListener#onPrebidNativeNotValid()} (PrebidNativeAd)} when Cached Ad is found and but is invalid
+     * and {@link PrebidNativeAdListener#onPrebidNativeNotFound()} ()} (PrebidNativeAd)} when Cached Ad is not found
+     */
+    public static void iterateWebViewListAsync(final List<WebView> webViewList, final int index, final PrebidNativeAdListener listener) {
```

Comment:
Why did you expose `iterateWebViewListAsync` ? It should be a part of open SDK API ?

Reply:
Restricted it to `default` visibility.